reflex/utils/exec.py

```diff
@@ -108,7 +108,14 @@ def run_process_and_launch_url(run_command: list[str], backend_present=True):
                         url = match.group(1)
                         if get_config().frontend_path != "":
                             url = urljoin(url, get_config().frontend_path)
-                        console.print(f"App running at: [bold green]{url}")
+                        if backend_present:
+                            console.print(
```

Comment:
nit: maybe put the backend url on a newline. Also remove the ending `)` from the green in the url, or it appears to be part of the url